spec/datadog/tracing/contrib/elasticsearch/support/client.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.shared_context 'Elasticsearch client' do
```

Comment:
This helper is simply extracting behaviour from `spec/datadog/tracing/contrib/elasticsearch/patcher_spec.rb` into its own helper file.